### PR TITLE
EPMRPP-46987 Launch logs reporting

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: src/realTimeReporter/reportingApi/attachData.ts [hunk 1]
```diff
@@ -5,38 +5,62 @@ import { AttachmentRQ, Attribute, LogRQ } from '../../models';
 
 type LogMessage = Pick<LogRQ, 'message'>;
 
+interface SendLogInterface {
+    info(message: LogMessage): void;
+    debug(message: LogMessage): void;
+    error(message: LogMessage): void;
+    trace(message: LogMessage): void;
+    fatal(message: LogMessage): void;
```
Comment (ElenaRomanchuk) [inline src/realTimeReporter/reportingApi/attachData.ts]: Also please add method for warnings.

Comment (AmsterGet) [reply src/realTimeReporter/reportingApi/attachData.ts]: Sure

File: src/realTimeReporter/reportingApi/attachData.ts [hunk 2]
```diff
@@ -1,42 +1,71 @@
...
     addDescription: PublicReportingAPI.addDescription,
-    addAttachment: PublicReportingAPI.addAttachment,
 
-    addLog: PublicReportingAPI.addLog,
-    logInfo: (message) => {
-        PublicReportingAPI.addLog(LOG_LEVELS.INFO, message);
-    },
-    logDebug: (message) => {
-        PublicReportingAPI.addLog(LOG_LEVELS.DEBUG, message);
+    sendLog: {
```
Comment (ElenaRomanchuk) [inline src/realTimeReporter/reportingApi/attachData.ts]: `sendLog` expected to be a function, not an object. It seems to me we should think if it's really necessery to add an additional object for logging methods.

Comment (AmsterGet) [reply src/realTimeReporter/reportingApi/attachData.ts]: Renamed to `log/launchLog`.
Readme file for the whole agent with two possible reporters will be updated in the scope of task for postFactum reporter.

**Pull request merged**